src/libraries/Microsoft.PowerFx.Core/Utils/TextFormatUtils.cs

```diff
@@ -2,15 +2,147 @@
 // Licensed under the MIT license.
 
 using System;
+using System.Collections.Immutable;
+using System.Globalization;
 using System.Text.RegularExpressions;
+using System.Xml;
 
 namespace Microsoft.PowerFx.Core.Utils
 {
+    /// <summary>
+    /// Definition for format string object ([$-FormatCultureName]FormatArg).
+    /// </summary>
+    internal class TextFormatArgs
+    {
+        /// <summary>
+        /// Culture name of the format string.
+        /// </summary>
+        internal string FormatCultureName { get; set; }
```

Comment:
>internal [](http://example.com/codeflow?start=8&length=8)

super nit: these can be public. The class is already internal, we can have public members on internal class.

Reply:
updated